common/lib/xmodule/xmodule/modulestore/xml_importer.py

```diff
@@ -356,7 +356,8 @@ def make_asset_id(course_id, asset_xml):
                     asset_md.from_xml(asset)
                     all_assets.append(asset_md)
         except OSError:
-            logging.error(f'Course import {course_id}: No {assets_filename} file present.')
+            # file does not exist.
+            logging.info(f'Course import {course_id}: No {assets_filename} file present.')
```

Comment:
Similarly here, it's not an error when the `asset.xml` file is not available in the tarball.